platformtour/templates/platformtour.html

```diff
@@ -0,0 +1,13 @@
+<article>
+    <h2 class="problem-header">{{ display_name }}</h2>
+    <p>{{ intro }}</p>
+
+    <button class="navmaker">{{ button_label }}</button>
+
+    <div class="sr">
+        <h4>Screen Reader Accessible Version of the Platform Tour</h4>
+        <ol id="screen-reader-steps"></ol>
```

Comment:
Does this need to be an ID?
There can now be multiple instantiations of the xblock on a single page..